packages/kbn-visualization-ui-components/components/chart_switch_trigger.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
```

Comment:
@MichaelMarcialis 

> The padding reductions made to the accordion contents feels odd. Now the accordion contents are no longer properly aligned with other content in the flyout. If this was done to address the fact that excessive padding was previously appearing to the right of the scrollbar, I think we should try an alternative approach. The reason this padding was appearing to the right was because a parent element of the scrolling container was receiving that padding. Instead, applying the padding to the scrolling container itself (or an element within it) should correctly address the issue without adversely affecting the alignment with other flyout content. Thoughts?

I reverted my changes in one PR and added what I think you mean in the next one. Please let me know if I understood you correctly 😊 

<img width="384" alt="Screenshot 2024-03-07 at 15 50 03" src="https://github.com/elastic/kibana/assets/4283304/a865ac47-eb2c-4fe3-bc8a-7a5269a753f6">

Reply:
Exactly! That looks wonderful. Thank you so much for making that change.